modmesh/pilot/_burgers1d.py

```diff
@@ -0,0 +1,260 @@
+import sys
```

Comment:
You should add a copyright comment block like other files.  Please do it in the next PR.